pages/home/home.ejs

```diff
@@ -2,108 +2,512 @@
+        <% if (courses && courses.length > 0) { %>
+        <div class="card mb-4">
+          <div class="card-header bg-primary text-white">Courses</div>
+          <table class="table table-sm table-hover table-striped">
+            <tbody>
+              <% courses.forEach(function(course) { %>
+              <tr>
+                <td>
+                  <a href="<%= plainUrlPrefix %>/course/<%= course.id %>">
+                    <%= course.label %>
+                  </a>
+                </td>
+              </tr>
+              <% }); %>
+            </tbody>
+          </table>
         </div>
+        <% } %> <!-- courses.length -->
 
-        <% if (course_instances && course_instances.length == 0) { %>
+        <div class="card mb-4">
+          <div class="card-header bg-primary text-white d-flex align-items-center">
+            <% if (courses && courses.length > 0) { %>
+            Course instances
+            <% } else { %>
+            Courses
+            <% } %>
+            <% if (authn_provider_name !== 'LTI') { %>
+            <a href="<%= plainUrlPrefix %>/enroll" class="btn btn-light btn-sm ml-auto">
+              <i class="fa fa-edit" aria-hidden="true"></i>
+              <span class="d-none d-sm-inline">Add or remove courses</span>
+            </a>
+            <% } %>
+          </div>
 
-        <% if (devMode) { %>
-        <div class="card-body">
-          No courses loaded. Click <strong>“Load from disk”</strong>
-          above and then click <strong>“Home”</strong> to come back to
-          this page.
+          <% if (course_instances && course_instances.length == 0) { %>
+          <% if (devMode) { %>
+          <div class="card-body">
+            No courses loaded. Click <strong>“Load from disk”</strong>
+            above and then click <strong>“Home”</strong> to come back to
+            this page.
+          </div>
+          <% } else { %> <!-- devMode -->
+          <div class="card-body">
+            No courses found. Use the “Add or remove courses” button to add one.
+          </div>
+          <% } %> <!-- devMode -->
+          <% } else { %> <!-- course_instances.length -->
+          <table class="table table-sm table-hover table-striped">
+            <tbody>
+              <% course_instances.forEach(function(course_instance) { %>
+              <tr>
+                <td>
+                  <a href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>">
+                    <%= course_instance.label %>
+                  </a>
+                </td>
+              </tr>
+              <% }); %>
+            </tbody>
+          </table>
+          <% } %> <!-- course_instances.length -->
         </div>
-        <% } else { %> <!-- devMode -->
-        <div class="card-body">
-          No courses found. Use the “Add or remove courses” button to add one.
-        </div>
-        <% } %> <!-- devMode -->
+      </div>
+      <% } %> <!-- isAuthenticated -->
+
```

Comment:
Maybe guard this code block with `!isAuthenticated` as students should be focused on only selecting/enrolling in their course vs. looking at PL capabilities.